components/_global-navigation.scss

```diff
@@ -57,10 +59,52 @@ $nav-width: 1024px;
 		}
 	}
 
+	&__icon-user {
+		border: solid 2px $color-dark-blue-gray;
+		border-radius: 50%;
+		fill: $color-dark-blue-gray;
+		height: 28px;
+		margin-right: 2px;
+		width: 28px;
+	}
+
+	&__notifications {
+		position: relative;
+	}
+
+	&__notification-counter-wrapper {
+		background-color: $color-tv;
+		border: 1px solid $color-white;
+		border-radius: $notification-counter-height / 2;
+		color: $color-white;
+		font-size: $typescale-size-minus-3;
+		height: $notification-counter-height - 2 * $notification-counter-border-width;
+		left: 6px;
+		line-height: $notification-counter-height - 2 * $notification-counter-border-width;
+		min-width: $notification-counter-height - 2 * $notification-counter-border-width;
```

Comment:
while `$notification-counter-height` is used for `width` calculations I'd name it differently: eg `$notification-counter-size`

BTW: we perform the same calculation 3 time, maybe it's time for a variable or changing `box-sizing` attribute for this component.